### ISSUE TITLE
Touchscreen not working in TWRP for for USC S6 SM-G920R4

### ISSUE THREAD

**Issue by stillrapids** [TeamWin/Team-Win-Recovery-Project#417]
Hello,

When I got my s6 in mid-April, I immediately flashed the zeroflte recovery on my SM-G920R4. However, after installing 2.8, the phone gets stuck in a sort of recovery boot loop. While the phone does indeed boot to recovery, it does not recognize touches inside of TWRP. Turn the phone off and it will boot to recovery again. The only workaround I've found is to reflash cf autoroot.

Based on what I've seen on teamuscellular.com, this is happening to everyone with a USC S6. Any news or suggestions?

**Comment by PolishedGarbage** [comment]
To our luck, it is still that way with even TWRP's current version. I tested it today and ended up doing a full wipe and re-flashing stock... I guess we will just have to hold off for a while. Sucks man

**Comment by freeklatin** [comment]
What was the software you guys used to flash back? Im currently bricked. TWRP is locked up and i cant use it, and when I try and flash factory firmware it fails in odin. Thinking its due to being on 5.1.1 and trying to downvert to 5.0.2. Just wondering what you used? Im lost and need my phone back lol.

**Comment by shaggyskunk** [comment]
Pretty sure that you can't go back to 5.0.2 - I'd try flashing the stock
firmware with Odin again - Are you using the latest version of Odin? It's
posted on sammobile.com

**Comment by freeklatin** [comment]
I am currently downloading the latestest. I was following a different Tut that had me download an older one i believe. The whole problem started when i downloaded the TWRP app tried to install, it restarted my phone, puts me into recovery with not touchscreen capabilities. Very frustrating. I can at least get it into download mode so i still have some hope that it can be fixed.

**Comment by shaggyskunk** [comment]
Did you use Odin to flash the latest version of TWRP? - That's the best way
to go

**Comment by freeklatin** [comment]
Yeah, i did that too, try and boot into recovery, takes about 30 seconds, goes into TWRP but i have no touchscreen capabilities. Its the only way i can get the screen to shut off. Otherwise its download mode

**Comment by freeklatin** [comment]
Thats exactly what the issue was. I was trying to push an out of date update from android rootz video. It must have been a 5.0.2 version. 5.1.1 seems to of worked. Glad to see progress on the situation.

**Comment by freeklatin** [comment]
Thank you for the prompt help and insight by the way shaggy. I appreciate it!

**Comment by shaggyskunk** [comment]
Happy that you got it sorted out :)

**Comment by PolishedGarbage** [comment]
## Still need any help? The process is very fresh in my head from a few days ago.

Im currently away from home but I have a kernel you can flash if you'd like root once again, it isnt SElinux enforcing so it allows root. You see on 5.1.1 you cant PingPongRoot it like you could on android 5.0.1-2.
If you interested lemme know!

**Comment by jimmyblaze1234** [comment]
What I've done is use a program on my PC called 'Wondershare, Dr. Fone for Android'. Anytime my phone keeps booting into recovery(stock), I start up this program on my PC, connect my Galaxy Core Prime SM-G360T, and once connected it gives me the option to "Exit Download Mode". So once I click that, my phone reboots normally into Android 5.1 Lollipop. I finally just installed TWRP 3.0.2-0, and when I boot into TWRP, the touchscreen is unresponsive. So, I just rebooted into Odin Mode, used Wondershare Dr. Fone, pulled outta Odin Mode, and my phone booted up like normal. I know this thread is old, but maybe it'll help someone out there. Please let me know.... Thanks...

**Comment by ghost** [comment]
Is this problem present with the latest version of TWRP?

**Comment by seasnail4** [comment]
Hey hopefully this isnt dead but I just installed twrp on my us Cellular galaxy s6 planning on installing /e/. I am also stuck here at twrp, screen unresponsive, cannot follow these directions:

because I cannot touch and slide. I have been trying to work with adb shell commands but seems twrp uses special commands. I also cannot reboot regularly. Any help or fixes yet?

**Comment by TheMoves** [comment]
There is a TWRP for the USC version of the S6...however there are no current ROMs. If you want to flash Android 10 and not have touch screen, then you need to flash one of the other TWRPs for the S6 like `zerofltexx`
